ecosystem/gstreamer_plugin/gst_mtl_common.c

```diff
@@ -391,3 +393,26 @@ gboolean gst_mtl_common_parse_dev_arguments(struct mtl_init_params* mtl_init_par
   gst_mtl_port_idx++;
   return ret;
 }
+
+mtl_handle gst_mtl_common_mtl_init(struct mtl_init_params *p, StDevArgs *devArgs, guint *log_level) {
```

Comment:
why log_level is passed by a pointer if it is not modified and it is just small type `int`?